pytorch_lightning/strategies/ddp.py

```diff
@@ -231,23 +236,17 @@ def _register_ddp_hooks(self) -> None:
                 import torch.distributed.algorithms.ddp_comm_hooks.post_localSGD_hook as post_localSGD
 
                 if isinstance(self._ddp_comm_state, post_localSGD.PostLocalSGDState):
-                    self._reinit_optimizers_with_post_localSGD(self._ddp_comm_state.start_localSGD_iter)
+                    self._enable_model_averaging()
 
-    def _reinit_optimizers_with_post_localSGD(self, warmup_steps: int):
+    def _enable_model_averaging(self):
```

Comment:
add a typehint for the return value
```suggestion
    def _enable_model_averaging(self) -> None:
```